tests/conftest.py

```diff
@@ -65,11 +66,15 @@ def html_tree(self, *path) -> str:
 
 
 @pytest.fixture()
-def sphinx_build_factory(make_app: Callable, tmp_path: Path) -> Callable:
+def sphinx_build_factory(make_app: Callable, tmp_path: Path, request) -> Callable:
     """Return a factory builder pointing to the tmp directory."""
 
-    def _func(src_folder: Path, **kwargs) -> SphinxBuild:
+    def _func(src_folder: str, **kwargs) -> SphinxBuild:
         """Create the Sphinxbuild from the source folder."""
+        no_temp = environ.get("PST_TEST_HTML_DIR", None)
```

Comment:
Nitpick: returning `None` is the default if the var isn't found. I am always conflicted on whether shorter/simpler (omitting None) is better in such contexts, or whether explicitness (including None) should be preferred. I guess really what I'm asking here is "what is your reason for including it here?"

```suggestion
        no_temp = environ.get("PST_TEST_HTML_DIR")
```

Reply:
I just never remember if `environ.get` raises or default to `None`/`''` when the env variable is not set, so I was explicit, but what you suggest also works for me.